pkg/commands/compute/init.go

```diff
@@ -944,7 +944,23 @@ func (c *InitCommand) FetchPackageTemplate(branch, tag string, archives []file.A
 		return err
 	}
 
-	filename := filepath.Base(c.CloneFrom)
+	tempdir, err := tempDir("package-init-download")
+	if err != nil {
```

Comment:
```suggestion
	if tempdir, err := tempDir("package-init-download"); err != nil {
```

Reply:
I had to revert this one because `tempdir` is used later (outside the scope of this `if` block), so it has to stay in scope.